package.json

```
@@ -23,7 +23,7 @@
     ]
   },
   "dependencies": {
-    "@sats-connect/core": "0.0.9",
+    "@sats-connect/core": "^0.0.11",
```

should use --save-exact
```suggestion
    "@sats-connect/core": "0.0.11",
```